.gitignore

```diff
@@ -0,0 +1,5 @@
+.idea
```

Comment:
This file should not be part of this pull request.

:bulb: Before creating a new branch to work on, make sure you're on the most recent version of `master`.